src/formElementPlugins/search/src/ApplicationView.tsx

```diff
@@ -123,20 +123,27 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
         <Markdown text={label} semanticComponent="noParagraph" />
       </label>
       <Markdown text={description} />
-      <Search
-        value={searchText}
-        loading={loading}
-        onSearchChange={handleChange}
-        onResultSelect={handleSelect}
-        minCharacters={minCharacters}
-        placeholder={placeholder}
-        results={
-          loading ? [{ title: strings.MESSAGE_LOADING }] : createResultsArray(results, resultFormat)
-        }
-        disabled={!isEditable}
-        input={{ icon: icon, iconPosition: 'left' }}
-        noResultsMessage={strings.MESSAGE_NO_RESULTS}
-      />
+      <Form.Field key={`search-${label}`} error={!validationState.isValid}>
```

Comment:
Does it have to be wrapped in `<Form.Field>`? There is already a Form.Field wrapper in the ApplicationViewWrapper, which is why most of the plugin components are just wrapped in a `<>` fragment.

You might not be able to avoid it, but if we can avoid `Form.Field`s inside `Form.Fields`, that'd be preferable. 
(I'll check in the morning if it's actually valid HTML)